version/version.go

```diff
@@ -15,6 +15,15 @@ limitations under the License.
 */
 package version
 
+import "fmt"
+
 var (
-	Version = "v0.0.0"
+	// This variable shows the commit hash of the repo so they can confirm they are on latest or specific version of the branch.
+	GitHash string
+	// This variable is dependent on what the current release is e.g. if it is v0.2.3 then this variable, outside of releases, will be v0.2.4-dev .
+	Version = "v0.2.4-dev"
 )
+
+func init() {
+	Version = fmt.Sprintf("%s (%s)", Version, GitHash)
```

Comment:
I would not update `Version`. Leave that as it is. Instead in `cmd/version.go`

```go
func versionCommand() *cobra.Command {                             
    versionCmd := &cobra.Command{                                  
        Use:   "version",                                          
        Short: "Print the version number of kuadrantctl",          
        Long:  "Print the version number of kuadrantctl",          
        RunE: func(cmd *cobra.Command, args []string) error {      
            err := utils.SetupScheme()                             
            if err != nil {                                        
                return err                                         
            }                                                      
                                                                   
            fmt.Printf("kuadrantctl %s (%s)\n", version.Version, version.GitHash)            
            return nil                                             
        },                                                         
    }                                                              
    return versionCmd                                              
}                                                                  

```

The formatting happens when somebody asks for the version, not for the variable value. 

It is not incorret what you did, so I will leave up to you.